scripts/testcase/tpl-testcase.sh

```diff
@@ -0,0 +1,33 @@
+# Template of testcase file
+# Each testcase file must have a format of "testcase-{suffix}.sh"
+
+# envs setup
+export KUBERNETES_SERVICE_HOST=kubernetes.default.svc
+export KUBERNETES_SERVICE_PORT=443
+
+export ENABLE_CALICO_NETWORK_POLICY=false
+export ENABLE_BANDWIDTH_PLUGIN=false
+export ENABLE_CILIUM_PLUGIN=false
+export ENABLE_MASQUERADE=false
+export ENABLE_IPV6=false
+
+export CNI_SPEC_TEMPLATE='{}'
+
+# Method to be invoked before install-cni.sh call
+# You can configure required mocks here
+function before_test() {
```

Comment:
It looks like the pattern of url matching in `curl` and `route` mocking is everywhere. Add them to the template?

Reply:
ok, added to template and moved route mock implementation to set of common mocks